src/components/DocumentViewer/DocumentViewerStyles.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import DocViewer from 'react-doc-viewer';
+
+export const DocumentViewerStyle = styled.div`
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+`;
+
+export const DocViewerStyle = styled(DocViewer)`
```

Comment:
Can you proxy the DocViewer props on this?
`styled(DocViewer)<Props>`

Reply:
done